ChopChop/ViewModels/OnlineRecipes/OnlineRecipeByUserViewModel.swift

```diff
@@ -48,10 +39,12 @@ class OnlineRecipeByUserViewModel: OnlineRecipeViewModel {
         storageManager.unrateRecipe(recipeId: recipe.id, rating: ownRating)
     }
 
-    private func creatorPublisher() -> AnyPublisher<User, Never> {
-        storageManager.userByIdPublisher(userId: recipe.userId)
-            .assertNoFailure()
-            .eraseToAnyPublisher()
+    func setShowRating(_ isShowing: Bool) {
+        isShowingRating = isShowing
     }
 
+    func toggleShowRating() {
+        isShowingRating.toggle()
+        self.objectWillChange.send()
```

Comment:
What's this for?

Reply:
I'm not sure why the view does not update without manually calling the update here